Tara/stations_challenge.py

```diff
@@ -0,0 +1,116 @@
+        print(f'{self.station_name} station is located at {self.location}')
+
+class SubwayStation(Station):
+    def__init__(self, station_name, location, lines):
+        super().__init__(station_name, location)
+        self.lines = lines 
+    def show_info(self):
+        print(f'Lines are {self.lines}, Station Name is {self.station_name},  Location is {self.location}")')   
+
+print('Question 2: Make an example subway station')
+'''
+Using your SubwayStation class, instantiate a subway station with the info below. 
+Then run the show_info() method to make sure you get the station_name, location, and lines printed out
+
+station_name: '14th street'
+location: '14th street and 7th avenue'
+lines: ['1', '2', '3', 'L']
+'''
+Station_1 = SubwayStation("14th Street","14th St & 7th Ave",['1','2','D','F'])
+Station_1.show_info()
+
+print('Question 3: Making the BusStation Class')
+
+'''
+Using the Station class below as the parent, make a child class called BusStation
+
+BusStation should:
+-have an additional attribute called 'routes' that is user-defined as a list during initialization. 
+    this will indicate where buses can go from this station. For example, ['DC', 'Philly', 'Pittsburgh']
+-have an additional attribute called 'open' that is always initalized as True (a boolean variable)
+-have additional methods called open_station() and close_station() to open and close the station
+-override the show_info() method from Station to display the bus routes and if the station is open, in addition to the station name and location
+'''
+class BusStation(Station):
+    def __init__(self, station_name, location, routes, open=True, hours="open"):
+        super().__init__(station_name, location)
+        self.routes = routes
+        self.open = open
+        self.hours = hours
+
+    def open_station(self):
+        self.open = True
+
+    def close_station(self):
+        self.open = False
+
+    def show_info(self):
+        if self.open == True:
+            self.hours = "open"
+        elif self.open == False:
+            self.hours = "closed"
+        print(f"Bus station name is {self.station_name}, located at {self.location}. Routes are {self.routes}, and is currently {self.hours}")
+
+Station_2 = BusStation("Penn Station","34th & 8th","Hudson River, New Jersey")
+
+Station_2.show_info()
+
+Station_2.close_station()
+
+Station_2.show_info()
+
+print('Question 4: Make an example bus station')
+'''
+Using your BusStation class, instantiate a bus station with the info below. 
+Then, run the show_info() method to make sure you get the station_name, location, routes, and whether the station is open printed out
+Then, demonstrate that you can close and open the bus station
+    i.e. that the show_info() method reflects correctly when the station is open versus closed
+
+station_name: 'NYC Megabus Stop'
+location: '34th street and 12th avenue'
+lines: ['Boston', 'DC', 'Philly']
+'''
+nyc_megabus = BusStation(station_name, location, lines)
```

Comment:
Here you need to give the parameters for BusStation, its asking you for an actual station name, location, and routes. for example ` SubwayStation("14th Street","14th St & 7th Ave",['1','2','D','F'])`